client/state/sharing/publicize/schema.js

```diff
@@ -0,0 +1,31 @@
+export const connectionsSchema = {
+	type: 'object',
+	patternProperties: {
+		'^\\d+$': {
+			type: 'object',
+			required: [ 'ID', 'site_ID' ],
+			properties: {
+				ID: { type: 'integer' },
+				site_ID: { type: 'integer' },
+				user_ID: { type: 'integer' },
+				keyring_connection_ID: { type: 'integer' },
+				keyring_connection_user_ID: { type: 'integer' },
+				shared: { type: 'boolean' },
+				service: { type: 'string' },
+				label: { type: 'string' },
+				issued: { type: 'string' },
+				expires: { type: 'string' },
+				external_ID: { type: 'string' },
+				external_name: { type: [ 'string', 'null' ] },
+				external_display: { type: 'string' },
+				external_profile_picture: { type: 'string' },
+				external_profile_URL: { type: 'string' },
+				external_follower_count: { type: 'integer' },
```

Comment:
Getting a schema validation warning here. This value is `null` in my case.

![image](https://cloud.githubusercontent.com/assets/1779930/13406766/4e6e1c86-def3-11e5-8f2c-88de3636b840.png)